cinema/serializers.py

```diff
@@ -85,9 +86,8 @@ class Meta:
 class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
-        Ticket.validate_ticket(
-            attrs["row"], attrs["seat"], attrs["movie_session"]
-        )
+        Ticket.validate_ticket(attrs["row"], attrs["seat"],
+                               attrs["movie_session"])
```

Comment:
The `validate_ticket` method call is missing the `error_to_raise` parameter. According to the `validate_ticket` method definition in the `Ticket` model, this parameter is required. You should pass `ValidationError` as the fourth argument to match the method signature.